streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -71,8 +72,8 @@
     private static final CompressionType COMPRESSION_TYPE = CompressionType.NO_COMPRESSION;
     private static final CompactionStyle COMPACTION_STYLE = CompactionStyle.UNIVERSAL;
     private static final long WRITE_BUFFER_SIZE = 16 * 1024 * 1024L;
-    private static final long BLOCK_CACHE_SIZE = 50 * 1024 * 1024L;
-    private static final long BLOCK_SIZE = 4096L;
+    protected static final long BLOCK_CACHE_SIZE = 50 * 1024 * 1024L;
```

Comment:
Why these two need to be protected now?

Reply:
oversight from previous change, I'll revert